.tekton/pull-request.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: tekton.dev/v1beta1
+kind: PipelineRun
+metadata:
+  annotations:
+    build.appstudio.redhat.com/commit_sha: '{{revision}}'
+    build.appstudio.redhat.com/pull_request_number: '{{pull_request_number}}'
+    build.appstudio.redhat.com/target_branch: '{{target_branch}}'
+    pipelinesascode.tekton.dev/max-keep-runs: "3"
+    pipelinesascode.tekton.dev/on-event: '[pull_request]'
+    pipelinesascode.tekton.dev/on-target-branch: '[main]'
+  name: contract-golden-image-on-pull-request
+spec:
+  params:
+    - name: git-url
+      value: '{{repo_url}}'
+    - name: revision
+      value: '{{revision}}'
+    - name: output-image
+      value: quay.io/redhat-appstudio/pull-request-builds:ec-golden-image-{{revision}}
+    - name: dockerfile
+      value: ./Containerfile
+    - name: path-context
+      value: .
+  pipelineRef:
+    name: docker-build
+    # TODO: Use replace git-commit tag with a floating tag so it can be managed by renovate bot.
+    bundle: quay.io/redhat-appstudio-tekton-catalog/pipeline-docker-build:165e81585fb7afd4fb17be0ef3851cb3962e281d@sha256:21e4124faa37485c8c170f3b367b4ae9334c1b6bc5420b802fbdcebd31069583
```

Comment:
this bundle has currently set hacbs=false, so you can enable it via param. This behavior will change soon probably.

Reply:
Thanks!